.github/workflows/release.yml

```diff
@@ -9,14 +9,28 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       nx_affected: ${{ steps.set-nx-affected.outputs.nx_affected }}
+    permissions:
```

Comment:
We need to add `permissions` for better supply chain security. We might need to experiment with this, but these permissions should be sufficient. Let me know if this needs to be changed!